crates/ark/src/modules/positron/package.R

```diff
@@ -21,6 +21,32 @@
 #' @export
 .ps.rpc.is_installed <- .ps.is_installed
 
+# Returns a list containing:
+#   * the version string if the package is installed and NULL otherwise
+#   * a minimum version to check for, with '0.0.0' as a dummy default
+#   * a logical indicating if package is installed at or above the minimum version
+#  This may seem weird, but it's impractical for positron-r to do version
+#  comparisons.
+#' @export
+.ps.rpc.packageVersion <- function(pkg, checkAgainst = "0.0.0") {
+    installed <- system.file(package = pkg) != ""
+
+    if (installed) {
+        version <- utils::packageVersion(pkg)
+        list(
+          version = as.character(version),
+          checkAgainst = as.character(checkAgainst),
```

Comment:
why is this argument returned to the caller?

Reply:
I guess it just felt convenient to package the version checked against and the result together? No more, no less. Maybe you're saying that either the returned info should be completist (include the package name) or be minimal (exclude the version checked against)?

Reply:
I think the idea is that the caller knows `checkAgainst` already, so no need to duplicate the info

Reply:
yep exactly